src/services/ExplorerService/fetcher.ts

```diff
@@ -162,6 +162,13 @@ export const apiFetcher = {
       },
     }),
 
+  fetchSearchByNameResult: (param: string) =>
+    v1Get<Response.Wrapper<UdtQueryResult, SearchResultType.UDT>[]>('udt_queries', {
```

Comment:
Consider using `v1GetUnwrappedPagedList`.

If you want to include the following type as part of the generic, you can add another generic parameter to the implementation of `v1GetUnwrappedPagedList`.